kedro_sagemaker/cli_functions.py

```diff
@@ -94,3 +96,18 @@ def write_file_and_confirm_overwrite(
         filepath.write_text(contents)
     elif on_denied_overwrite:
         on_denied_overwrite(filepath)
+
+
+def lookup_mlflow_run_id(context, sagemaker_execution_arn: str):
+    import mlflow
+    from kedro_mlflow.config.kedro_mlflow_config import KedroMlflowConfig
+
+    mlflow_conf: KedroMlflowConfig = context.mlflow
+    mlflow_run = mlflow.search_runs(
```

Comment:
Maybe we should add some verbose logging on lookup failures, to help debugging it if sth goes wrong

Reply:
Good idea, will do!